src/sqltest/platform/dialog/dialogPane.test.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
```

Comment:
These are important tests. If this change prevents us from testing things that rely on it then it's not worth making. Do you think we can find a way to keep these tests?